catalyst_voices/lib/pages/registration/recover/seed_phrase/account_details_panel.dart

```diff
@@ -0,0 +1,200 @@
+
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      children: [
+        const SizedBox(height: 24),
+        Text(
+          context.l10n.recoveryAccountTitle,
+          style: theme.textTheme.titleMedium?.copyWith(color: textColor),
+        ),
+        const SizedBox(height: 24),
+        Expanded(
+          child: SingleChildScrollView(
+            child: _BlocAccountSummery(
+              onRetry: () => unawaited(retryAccountRestore(context)),
+            ),
+          ),
+        ),
+        const SizedBox(height: 24),
+        const _BlocNavigation(),
+      ],
+    );
+  }
+
+  Future<void> retryAccountRestore(BuildContext context) async {
+    final recover = RegistrationCubit.of(context).recover;
+    await recover.recoverAccount();
+  }
+}
+
+class _BlocAccountSummery extends StatelessWidget {
+  final VoidCallback? onRetry;
+
+  const _BlocAccountSummery({
+    this.onRetry,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocRecoverBuilder<Result<AccountSummaryData, LocalizedException>?>(
+      selector: (state) => state.accountDetails,
+      builder: (context, state) {
+        return switch (state) {
+          Success<AccountSummaryData, LocalizedException>(:final value) =>
+            _RecoveredAccountSummary(
+              walletConnection: value.walletConnection,
+              walletSummary: value.walletSummary,
+            ),
+          Failure<AccountSummaryData, LocalizedException>(:final value) =>
+            _RecoverAccountFailure(
+              exception: value,
+              onRetry: onRetry,
+            ),
+          _ => const Center(child: VoicesCircularProgressIndicator()),
+        };
+      },
+    );
+  }
+}
+
+class _RecoveredAccountSummary extends StatelessWidget {
+  final WalletConnectionData walletConnection;
+  final WalletSummaryData walletSummary;
+
+  const _RecoveredAccountSummary({
+    required this.walletConnection,
+    required this.walletSummary,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      mainAxisSize: MainAxisSize.min,
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      children: [
+        WalletConnectionStatus(
+          icon: walletConnection.icon,
+          name: walletConnection.name,
+          isConnected: walletConnection.isConnected,
+        ),
+        const SizedBox(height: 8),
+        const _RecoverStatusText(),
+        const SizedBox(height: 24),
+        WalletSummary(
+          balance: walletSummary.balance,
+          address: walletSummary.address,
+          clipboardAddress: walletSummary.clipboardAddress,
+        ),
+      ],
+    );
+  }
+}
+
+class _RecoverAccountFailure extends StatelessWidget {
+  final LocalizedException exception;
+  final VoidCallback? onRetry;
+
+  const _RecoverAccountFailure({
+    required this.exception,
+    this.onRetry,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
```

Comment:
Column with a single child shouldn't be needed, if you want to stretch you can use SizedBox with infinite width or other patterns

Reply:
yeah i had more widgets in there but later decided to remove them but Column stayed